typescript/api/services/RecordsService.ts

```diff
@@ -265,6 +266,10 @@ export module Services {
       return this.storageService.getMeta(oid);
     }
 
+    getRecordAudit(oid: any): Promise < any > {
```

Comment:
Once you've done the change for https://github.com/redbox-mint/sails-hook-redbox-storage-mongo/pull/2
adding the date parameters here should be simple.